Assets/Python/Afforess/AutomatedSettings.py

```
@@ -1,195 +1,268 @@
-	CyMessageControl().sendModNetMessage(MODDEROPTION_EVENT_ID, GC.getGame().getActivePlayer(), int(ModderOptionTypes.MODDEROPTION_HIDE_AUTO_PROMOTE), int(value), 0)
+    _setPlayerOption(_MOD_OPT_HIDE_AUTO_PROMOTE, value)
 
 
 def setXMLOptionsfromIniFile():
-	print "Initializing Automated Settings"
-	AutomatedOptions = BugOptions.getOptions("AutomatedSettings").options
-	for i in range(len(AutomatedOptions)):
-		if AutomatedOptions[i].getKey() != "Reset":
-			AutomatedOptions[i].doDirties()
+    """Optimized initialization of automated settings"""
+    print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)